src/ui/filter/readonly.go

```diff
@@ -42,6 +42,7 @@ func filter(req *http.Request, resp http.ResponseWriter) {
 	}
 	if matchRepoTagDelete(req) {
 		resp.WriteHeader(http.StatusServiceUnavailable)
```

Comment:
Should it be 403?

Reply:
actually, 403 is a workaround here. Docker defined several DoNoRetry errors, 403 is one of them, and no 503 in.

Reply:
yep, conventionally when the system set to a mode that some features are disabled, the error should be `503`